manifests/opsmanager.pp

```diff
@@ -0,0 +1,59 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::params::opsmanager_user,
+  String[1] $group                               = $mongodb::params::opsmanager_group,
+  Variant[Boolean, String[1]] $ensure            = $mongodb::params::opsmanager_ensure,
```

Comment:
Please allow only the approriate strings for `$ensure`, probably something like this:
`Enum['present', 'latest', 'absent']`

Reply:
A string can also be used to ensure an exact version.